authz/providers/azure/rbac/rbac.go

```diff
@@ -180,6 +184,8 @@ func newAccessInfo(tokenProvider graph.TokenProvider, rbacURL *url.URL, opts aut
 		useNamespaceResourceScopeFormat:        opts.UseNamespaceResourceScopeFormat,
 		httpClientRetryCount:                   authopts.HttpClientRetryCount,
 		auditSAR:                               opts.AuditSAR,
+		runtimeConfigPath:                      opts.RunTimeConfigPath,
+		runtimeConfigReader:                    os.ReadFile,
```

Comment:
can we have a strongly typed config? without a type the config appears to be opaque. it will make maintenance harder.